spec/support/fixtures/test/main.swift

```diff
@@ -1,7 +1,7 @@
 protocol RedundantProtocol {
 }
 
-class SomeClass: RedundantProtocol {
+public class SomeClass: RedundantProtocol {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/manicmaniac/danger-periphery/blob/529aa7e414601b50d75108ce9802e8359e274382/spec/support/fixtures/test/main.swift#L4"></span>Class 'SomeClass' is declared public, but not used outside of the module</td>
    </tr>
  </tbody>
</table>